app/views.py

```diff
@@ -36,5 +40,17 @@ def terms_and_conditions(request):
     return render(request, 'terms_and_conditions.html')
 
 
+def protectedMedia(request, file):
```

Comment:
Don't use `file` its a reserved name from python. Use `f` or `file_` instead.

Reply:
Got it, thank-you.